pool/web/view/page/page_admin_contact.ml

```diff
@@ -313,18 +313,23 @@ let index Pool_context.{ language; _ } contacts query =
     CCList.map row contacts
   in
   let target_id = "contacts-list" in
+  Component.List.create
```

Comment:
In `let rows =`, make sure to wrap the email address in the `email_with_icons` function (same for the admin table)

Reply:
And also the background color